tests/unit_tests/simtel/test_simtel_histograms.py

```diff
@@ -65,3 +47,55 @@ def test_export_histograms(simtel_array_histograms_instance, io_handler):
     assert len(list_of_tables) == 144
     for table in list_of_tables:
         assert isinstance(table, Table)
+
+
+def test_number_of_histograms(simtel_array_histograms_file, simtel_array_histograms_instance):
+    instance_alone = SimtelHistograms(histogram_files=simtel_array_histograms_file, test=True)
+    # If combined_hists is None
+    assert instance_alone.combined_hists is None
+    assert instance_alone.number_of_histograms == 145
+    assert (
+        len(simtel_array_histograms_instance.combined_hists)
+        == simtel_array_histograms_instance.number_of_histograms
+    )
+
+
+def test_get_histogram_title(simtel_array_histograms_file, simtel_array_histograms_instance):
+    instance_alone = SimtelHistograms(histogram_files=simtel_array_histograms_file, test=True)
+    # If combined_hists is None
+    assert instance_alone.combined_hists is None
+    assert instance_alone.number_of_histograms == 145
+    for instance in [instance_alone, simtel_array_histograms_instance]:
+        assert instance.get_histogram_title(0) == "Events, without weights (Ra, log10(E))"
+
+
+def test_combine_histogram_files(simtel_array_histograms_file):
+    # Reading one histogram file
+    instance_alone = SimtelHistograms(histogram_files=simtel_array_histograms_file, test=True)
+
+    instance_alone.combine_histogram_files()
+
+    # Passing the same file twice
+    instance_all = SimtelHistograms(
+        histogram_files=[simtel_array_histograms_file, simtel_array_histograms_file], test=True
+    )
+    instance_all.combine_histogram_files()
```

Comment:
Not related to this PR, but I am curious, this function is essentially equivalent to the `add_histogram` tool that Konrad mention in his e-mail yesterday, right?